Simulator/Edges/AllEdges.cpp

```diff
@@ -109,22 +81,16 @@ void AllEdges::setupEdges(const int numVertices, const int maxEdges)
    countVertices_ = numVertices;
 
    if (maxTotalEdges != 0) {
-      destVertexIndex_ = new int[maxTotalEdges];
-      W_ = new BGFLOAT[maxTotalEdges];
-      summationPoint_ = new BGFLOAT *[maxTotalEdges];
-      sourceVertexIndex_ = new int[maxTotalEdges];
-      type_ = new edgeType[maxTotalEdges];
-      inUse_ = new bool[maxTotalEdges];
-      edgeCounts_ = new BGSIZE[numVertices];
+      destVertexIndex_.resize(maxTotalEdges);
+      W_.resize(maxTotalEdges, 0);
+      summationPoint_.resize(maxTotalEdges);
+      sourceVertexIndex_.resize(maxTotalEdges);
+      type_.resize(maxTotalEdges);
+      inUse_ = make_unique<bool[]>(maxTotalEdges);
+      edgeCounts_.resize(numVertices, 0);
```

Comment:
Why are some initialized and some not?

Reply:
All member variables are now initialized.